mne/viz/tests/test_3d.py

```diff
@@ -185,6 +187,21 @@ def test_plot_evoked_field(renderer):
             )
         evoked.plot_field(maps, time=0.1, n_contours=n_contours)
 
+    # Test plotting inside an existing Brain figure. This should not work in a notebook.
+    brain = Brain("fsaverage", "lh", "inflated", subjects_dir=subjects_dir)
+    if get_3d_backend() == "notebook":
```

Comment:
I thought that `renderer` ran both notebook and non-notebook variants, but it doesn't look that way :(

Looks like we'd have to create a `test_notebook.py` like this to test this stuff properly on notebook, basically each "test" contains Python code that must be self-contained (with all imports and variables etc.) necessary to run it in `nbexec`:

https://github.com/mne-tools/mne-python/blob/main/mne/viz/_brain/tests/test_notebook.py

One fairly low-effort solution is to write some minimal code to take care of covering the `time_viewer=True` and `time_viewer=False` cases plus the Brain-embedding case (which would cover this `NotImplementedError`. If you're sick of this notebook business I could push it quickly I think.